### DISCUSSION TITLE
Could Pinia work with Reactjs/React native ?

### DISCUSSION THREAD

**Discussion by tidan-16** [vuejs/pinia#1883] [Ideas]
Hi, I am making research on react native to build a good mobile application. The only problem is, with react, Redux and other store management librairies are just too hard to work with for nothing. (In my opinion)

That being said, could Pinia work with Reactjs/React native ? Right now it seems to not work together from what ive seen and read. 

I think pinia with React could be a good idea.

 I cant see a discussion or an issue talking about this so thats why im asking it in here. Pinia is easy to work with.

**Comment by abhishekZom** [comment]
No, Pinia is designed specifically for Vue.

For React you can go with redux/mobx or useState for local state or the built-in context api.

**Reply by abhishekZom** [reply]
Also, if you take a closer look into the latest redux approach i.e., redux-toolkit, you will realise its pretty similar to Pinia.

**Comment by posva** [comment]
Pinia relies on Vue's reactivity system so unless you also use `@vue/reactivity` within your React app (which maybe there is a project doing), you won't be able to use pinia with React
